src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -0,0 +1,257 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import PrimaryToolbar from '@redhat-cloud-services/frontend-components/PrimaryToolbar';
+import { conditionalFilterType } from '@redhat-cloud-services/frontend-components/ConditionalFilter/conditionalFilterConstants';
+import { EmptyTable } from '@redhat-cloud-services/frontend-components/EmptyTable';
+import { TableToolbar } from '@redhat-cloud-services/frontend-components/TableToolbar';
+
+import { global_success_color_100 as globalSuccessColor100 } from '@patternfly/react-tokens/dist/js/global_success_color_100';
+import { global_danger_color_100 as globalDangerColor100 } from '@patternfly/react-tokens/dist/js/global_danger_color_100';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import { Card, CardBody } from '@patternfly/react-core/dist/js/components/Card';
+import { sortable } from '@patternfly/react-table/dist/js/components/Table/utils/decorators/sortable';
+import { Table } from '@patternfly/react-table/dist/js/components/Table/Table';
+import { TableBody } from '@patternfly/react-table/dist/js/components/Table/Body';
+import { TableHeader } from '@patternfly/react-table/dist/js/components/Table/Header';
+import { Bullseye } from '@patternfly/react-core/dist/js/layouts/Bullseye';
+import {
+  EmptyState,
+  EmptyStateVariant,
+} from '@patternfly/react-core/dist/js/components/EmptyState/EmptyState';
+import { EmptyStateBody } from '@patternfly/react-core/dist/js/components/EmptyState';
+import {
+  Pagination,
+  PaginationVariant,
+} from '@patternfly/react-core/dist/js/components/Pagination/Pagination';
+import { Title } from '@patternfly/react-core/dist/js/components/Title';
+
+import messages from '../../Messages';
+import MessageState from '../MessageState/MessageState';
+import Loading from '../Loading/Loading';
+
+const AffectedClustersTable = ({ affectedClusters }) => {
+  const intl = useIntl();
+  const { isError, isUninitialized, isLoading, isFetching, isSuccess, data } =
```

Comment:
consider something like this...

`   const { isError, isUninitialized, isLoading, isFetching, isSuccess, data:rows = [] } =  affectedClusters;`

if yah do the thing mentioned in  src/Services/SmartProxy.js the above will save yah doing the line 40